### PR TITLE
Fixed extra table legs

### LINKED ISSUE #3544: Table icons messing up
Not sure if anyone else has noticed but it would seem table icons are a little broken. When you have like-tables merge into a larger one it works, but you still see the normal single table underneath it. Just the icon, not the object.
dased
This is much more visible with glass tables as you see the single table under the merged one

### PR DESCRIPTION
**Pull request by Tastyfish**
* Fixes #3544 
* Also moved a file that was horribly misnamed and in its own nearly-root folder.

### CODE REVIEW

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -42,7 +42,6 @@
 
 /obj/structure/table/update_icon()
 	if(smooth && !flipped)
-		icon_state = initial(icon_state)
```
Comment (Crazylemon64) [inline code/game/objects/structures/tables_racks.dm]: Are you sure this won't break if you flip and un-flip a table?

**Comment by Tastyfish** [comment]
@Crazylemon64 Somehow your comment is on the before side now (I think I broke github) and it won't let me comment on it, it was my mistake on the commit that it was just deleted, which I realized a minute after I submitted. Good catch though.

**Comment by Crazylemon64** [comment]
I deleted it, since github didn't clear it away as the comment was on removed lines, and not on added ones.

**Comment by Tastyfish** [comment]
Yeah, interesting to find a bug in github. Resolved either way.

**Pull request merged**